app/assets/javascripts/discourse/app/components/sidebar/panel-header.gjs

```diff
@@ -1,28 +1,29 @@
 import Component from "@glimmer/component";
 import { service } from "@ember/service";
-import { ADMIN_PANEL } from "discourse/lib/sidebar/panels";
 import BackToForum from "./back-to-forum";
 import Filter from "./filter";
+import FilterNoResults from "./filter-no-results";
 import ToggleAllSections from "./toggle-all-sections";
 
-export default class AdminHeader extends Component {
+export default class PanelHeader extends Component {
   @service sidebarState;
 
   get shouldDisplay() {
-    return this.sidebarState.isCurrentPanel(ADMIN_PANEL);
+    return this.sidebarState.currentPanel.displayHeader;
   }
 
   <template>
     {{#if this.shouldDisplay}}
-      <div class="sidebar-admin-header">
-        <div class="sidebar-admin-header__row">
+      <div class="sidebar-panel-header">
+        <div class="sidebar-panel-header__row">
           <BackToForum />
```

Comment:
Should we maybe `yield` here? I don't think `BackToForum` would apply in any case but admin sidebar. Kind of a short term problem as we are talking about "sidebar of sidebars", it just feels strange that this is hardcoded at the moment.

Reply:
I don't think `yield` would work here.

The sidebar is only rendered from one point in the application.

`yield` would work if we were reusing the component in multiple places.

Reply:
Also, this way I can reuse the Header as is in the doc sidebar TC, which is cleaner than reproducing the components and mount then in the `PluginOutlet`